dali/pipeline/operators/fused/normalize_permute.h

```diff
@@ -32,6 +32,11 @@ class NormalizePermute : public Operator<Backend> {
     H_(spec.GetArgument<int>("height")),
     W_(spec.GetArgument<int>("width")),
     C_(IsColor(spec.GetArgument<DALIImageType>("image_type")) ? 3 : 1) {
+    DALI_WARN("Warning: Deprecated: NormalizePermute: Use CropMirrorNormalize (without crop and "
```

Comment:
Too many colons here...
```suggestion
    DALI_WARN("Warning: NormalizePermute is deprecatred. Use CropMirrorNormalize (without crop and "
```